src/domain/challenge/ecoverse/ecoverse.dto.create.ts

```diff
@@ -0,0 +1,45 @@
+import { Field, InputType } from '@nestjs/graphql';
+import { IsOptional, MaxLength } from 'class-validator';
+import { SMALL_TEXT_LENGTH, TINY_TEXT_LENGTH } from '@src/common/constants';
+import { CreateContextInput } from '@domain/context/context';
+// import {
+//   IsUniqueTextId,
+//   TextIdType,
+// } from '@src/core/validation/constraints/unique.text.id';
+
+@InputType()
+export class CreateEcoverseInput {
+  @Field({
+    nullable: false,
+    description: 'The unique text ID for the ecoverse',
+  })
+  // @IsUniqueTextId(TextIdType.ecoverse, {
```

Comment:
Why is this commented out?

Reply:
See above comment - this decorator was breaking the generation / loading. And think will replace with custom Scalar so did not fight it.